### DISCUSSION TITLE
Sass @import not working in bootstrap-email.config.scss

### DISCUSSION THREAD

**Discussion by cernyjakub** [bootstrap-email/bootstrap-email#101] [Help]
Hello! I am trying to integrate the alpha version into my Rails app. I have added some custom tweaks into `boostrap-email.config.scss` file in my `app/assests/stylesheets` folder. 

All works fine, but when I tried to use sass `@import` to load my color variables from other file (used in web CSS style), system tells me `Error: File to import not found or unreadable`. The file is in the same folder - it seems to me Sass is not having that folder in its load path... Any advice?  I think having the option to re-use color definitions from normal web stylesheets would be very cool.

Thanks in advance
Jakub

**Comment by stuyam** [comment]
Yeah the idea is that you should be able to do that, though that is complicated specifying the load path. Im trying to think of the best way to handle that, maybe just load path the directory the config file is in? Hmm... would you mind opening this as an issue instead of a discussion so we can track that there? 👍
